server.js

```diff
@@ -47,18 +55,59 @@ app.get('/api', function apiIndex(req, res) {
   // It would be seriously overkill to save any of this to your database.
   // But you should change almost every line of this response.
   res.json({
-    woopsIForgotToDocumentAllMyEndpoints: true, // CHANGE ME ;)
-    message: "Welcome to my personal api! Here's what you need to know!",
-    documentationUrl: "https://github.com/example-username/express-personal-api/README.md", // CHANGE ME
-    baseUrl: "http://YOUR-APP-NAME.herokuapp.com", // CHANGE ME
+    woopsIForgotToDocumentAllMyEndpoints: false, // CHANGE ME ;) -done
```

Comment:
Since you have this documentation here, you aren't using the API controller in your controllers directory. It seems like you changed that code without understanding why you were changing it, or how it would fit into your overall app.